packages/server-wallet/src/wallet/index.ts

```diff
@@ -364,14 +408,29 @@ export class Wallet implements WalletInterface, ChainEventSubscriberInterface {
         {channelId}
       );
     };
-    const criticalCode: AppHandler<SingleChannelResult> = async (tx, channel) => {
-      const nextState = getOrThrow(CloseChannel.closeChannel(channel));
-      const {outgoing, channelResult} = await this.store.signState(channelId, nextState, tx);
 
-      return {outbox: outgoing.map(n => n.notice), channelResult};
+    const criticalCode: AppHandler<void> = async (tx, channel) => {
+      // TODO: (Objectives Rewrite) Keeping this here b/c we need to do input validation
+      // and check if its our turn and throw an error as existing tests expect
+      getOrThrow(CloseChannel.closeChannel(channel));
```

Comment:
This thing generates a "sign state action" which is unnecessary. We just need it for request validation cc @kerzhner